CHANGELOG.md

```diff
@@ -8,6 +8,109 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Manufacturing as a Service (MaaS) KIT](https://eclipse-tractusx.github.io/docs-kits/category/manufacturing-as-a-service-kit) |                                                                           n/a                                                                           |                                                                                                                             [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Manufacturing%20as%20a%20Service%20Kit/Manufacturing%20as%20a%20Service%20KIT%20Changelog)                                                                                                                             |
+| [Modular Production KIT](https://eclipse-tractusx.github.io/docs-kits/category/modular-production-kit)                        |                                                                           n/a                                                                           |                                                                                                                                         [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Modular%20Production%20Kit/Modular%20Production%20Kit%20Changelog)                                                                                                                                         |
+| [Policy Hub](https://github.com/eclipse-tractusx/policy-hub)                                                                  |                            policy-hub: [1.0.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.0.0)                            |                                                                                                                                                            [1.0.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.0.0)                                                                                                                                                             |
+| [Portal](https://github.com/eclipse-tractusx/portal)                                                                          |                                  portal: [2.0.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.0.0)                                  | Registration: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.0)<br/><br/>Frontend: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.0.0)<br/>Backend: [v2.0.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.0.0)<br/> Assets: [v2.0.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.0.0) |
```

Comment:
please change the `<br/><br/>` between Registration and Frontend to `<br/>`, it looks a bit weird

| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.0.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.0.0) | Registration: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.0)<br/>Frontend: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.0.0)<br/>Backend: [v2.0.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.0.0)<br/> Assets: [v2.0.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.0.0) |

Maybe we could add this time also a line for IAM (it's part of the release since the first one, but apparently newer mentionend in the changelog, I'd recommend to add  a row under the portal row:
| [IAM](https://github.com/eclipse-tractusx/portal-iam) | iam: [3.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/v3.0.0) | centralidp: [v3.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/centralidp-3.0.0)<br/>sharedidp: [v3.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/sharedidp-3.0.0)

Reply:
updated Portal. Beside that: Shall we add IAM as Portal IAM or just IAM?

Reply:
I'd call it IAM, as it's used by almost all components

Reply:
Added IAM as discussed as one component with 2 helm charts and app version = keycloak readme